rules/windows/credential_access_lsass_openprocess_api.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/03/02"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: Lsass access events added in Elastic Endpoint 8.7."
+min_stack_version = "8.7.0"
+updated_date = "2023/03/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies access attempts to LSASS handle, which may indicate an attempt to dump credentials from LSASS memory.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "LSASS Process Access via Windows API"
+note = """## Setup
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
+references = ["https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1003.001/T1003.001.md"]
+risk_score = 47
+rule_id = "ff4599cb-409f-4910-a239-52e4e6f532ff"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+api where Target.process.name : "lsass.exe"
```

```suggestion
api where host.os.type == "windows" and Target.process.name : "lsass.exe"
```
As per https://github.com/elastic/detection-rules/pull/2593